config/SOUE01/rels/d_a_obj_iron_stageNP/symbols.txt

```diff
@@ -2,15 +2,24 @@ _prolog = .text:0x00000000; // type:function size:0x2C scope:global
 _epilog = .text:0x00000030; // type:function size:0x2C scope:global
 _unresolved = .text:0x00000060; // type:function size:0x4 scope:global
 dAcOironStage_c_classInit__Fv = .text:0x00000070; // type:function size:0x64
-AcOironStage__initModels = .text:0x000000E0; // type:function size:0xF0
-AcOironStage__init = .text:0x000001D0; // type:function size:0xD0
-fn_362_2A0 = .text:0x000002A0; // type:function size:0x8
-fn_362_2B0 = .text:0x000002B0; // type:function size:0x8
-AcOironStage__draw = .text:0x000002C0; // type:function size:0x28
+createHeap__15dAcOironStage_cFv = .text:0x000000E0; // type:function size:0xF0
+create__15dAcOironStage_cFv = .text:0x000001D0; // type:function size:0xD0
+doDelete__15dAcOironStage_cFv = .text:0x000002A0; // type:function size:0x8
+actorExecute__15dAcOironStage_cFv = .text:0x000002B0; // type:function size:0x8
+draw__15dAcOironStage_cFv = .text:0x000002C0; // type:function size:0x28
 __dt__15dAcOironStage_cFv = .text:0x000002F0; // type:function size:0x74
 _ctors = .ctors:0x00000000; // type:label scope:global
 _dtors = .dtors:0x00000000; // type:label scope:global
-lbl_362_rodata_0 = .rodata:0x00000000; // type:object size:0x1C data:float
-g_profile_OBJ_IRON_STAGE = .data:0x00000000; // type:object size:0x44
-lbl_362_data_44 = .data:0x00000044; // type:object size:0x24
-AcOironStage__vtable = .data:0x00000068; // type:object size:0x80
+@14058 = .rodata:0x00000000; // type:object size:0x4 scope:local data:float
```

Comment:
Meticulously applying all these symbol names isn't really necessary, marking them scope:local is typically enough. The actual numbers will change due to unrelated header changes anyway. Nothing wrong with doing it though.